.github/workflows/test-application.yaml

```
@@ -193,3 +193,125 @@ jobs:
             - name: Build admin ui
               run: npm run build
               working-directory: ${{ matrix.working-directory }}/assets/admin
+
+    php-windows:
+        name: "PHP ${{ matrix.php-version }} on Windows (mysql ${{ matrix.mysql-version }}, ${{ matrix.phpcr-transport }}, ${{ matrix.dependency-versions }})"
```

dont have a strong opionion what to include here, but would like to make it consistent with the other job 🙂 

```suggestion
        name: "PHP ${{ matrix.php-version }} and Node.js ${{ matrix.node-version }} on Windows"

```